telos-frontend/src/components/todo/DashboardTodo.jsx

```diff
@@ -62,65 +64,38 @@ const outdated = {
   fontWeight: 'Bold',
 };
 
-const DashboardTodo = () => {
-  const listitems = [
-    {
-      name: 'isOverdue',
-      due: '2021-04-30',
-      isOverdue: true,
-      completed: false,
-    },
-    {
-      name: 'OutDated',
-      due: '2021-01-30',
-      isOverdue: false,
-      completed: true,
-    },
-    {
-      name: '一二三四五',
-      due: '2021-04-30',
-      isOverdue: true,
-      completed: true,
-    },
-    {
-      name: '上山打老虎',
-      due: '2021-04-30',
-      isOverdue: true,
-      completed: false,
-    },
-  ];
-
+const DashboardTodo = ({ date }) => {
   const classes = useStyles();
   const [todoName, setTodoName] = useState('');
-  const [todoDueDate, setTodoDueDate] = useState('2021-01-01');
-  const [newItem, setNewItem] = useState(listitems);
+  const [todoDueDate, setTodoDueDate] = useState(undefined);
+  const [todos, setTodos] = useState([]);
   const [anchorEl, setAnchorEl] = useState(null);
   const [cancel, setCancel] = useState([0]);
   const [reDate, setReDate] = useState('');
-  const [selectedTodo, setselectedTodo] = useState('');
+  const [selectedTodo, setSelectedTodo] = useState({});
   const [open, setOpen] = useState(false);
   const [migrate, setMigrate] = useState(false);
 
   function dateToTimestamp(endTime) {
```

Comment:
This method is a little confusing. A small comment explaining its purpose may be useful here.

Reply:
This is written by the previous team so I'm not really sure what does it do.